### PR TITLE
Update styling for `AvatarGroup`

### PR DESCRIPTION
**Pull request by toha993**

### CODE REVIEW

File: packages/react/src/avatar-group/AvatarGroup.tsx [hunk 1]
```diff
@@ -29,7 +29,12 @@ export const AvatarGroup = forwardRef<HTMLDivElement, AvatarGroupProps>(
         {overflowChildren.length > 0 && (
           <HoverCard>
             <HoverCardTrigger asChild>
-              <Avatar colorScheme="gray" size={size}>
+              <Avatar
+                bg="neutral.150"
+                color="bg.neutral.inverse"
```
Comment (radnan) [inline packages/react/src/avatar-group/AvatarGroup.tsx:34]: let's change the `neutral` avatar to have these styles rather than manually tweaking them from here

File: packages/react/src/avatar/Avatar.tsx [hunk 2]
```diff
@@ -54,10 +52,18 @@ export const Avatar = forwardRef<HTMLSpanElement, AvatarProps>(
           <Box asChild objectFit="cover" rounded="inherit" size="full">
             <RadixAvatar.Image alt={name} src={src} />
           </Box>
-          <Box asChild {...styles.fallback({ size })}>
+          <Box asChild {...styles.fallback({})}>
             <RadixAvatar.Fallback delayMs={FALLBACK_DELAY_IN_MS}>
               {/* TODO: Add a generic user icon, if `children` is `undefined` */}
-              {icon ? icon : name ? getInitialsFromName(name) : children}
+              {icon ? (
```
Comment (radnan) [inline packages/react/src/avatar/Avatar.tsx:58]: let's revert these changes from this PR? since the other avatar PR already has them - this PR should only have avatar group related changes

Comment (toha993) [reply packages/react/src/avatar/Avatar.tsx:58]: Vai, I rebased the branch with avatar design fix, as I needed the updated style. Do I not do it?

Comment (radnan) [reply packages/react/src/avatar/Avatar.tsx:58]: oops didn't check the commits - ignore this comment then

File: packages/react/src/avatar-group/AvatarGroup.tsx [hunk 3]
```diff
@@ -29,16 +38,31 @@ export const AvatarGroup = forwardRef<HTMLDivElement, AvatarGroupProps>(
...
-                </Box>
-              ))}
+            <HoverCardContent flexDirection="column" gap="xs">
+              {overflowChildren.map((child, index) =>
+                isValidElement<
+                  ComponentPropsWithRef<typeof Avatar> &
+                    ComponentPropsWithRef<typeof Tooltip>
+                >(child) ? (
+                  <Flex flexDirection="row" gap="4" key={index}>
+                    <AvatarGroupContext.Provider value={{ size }}>
```
Comment (radnan) [inline packages/react/src/avatar-group/AvatarGroup.tsx:52]: let's change the size to `md` for the popover
```suggestion
                    <AvatarGroupContext.Provider value="md">
```

**Review by radnan — APPROVED** [review]
